opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -946,7 +946,7 @@ public void prepareIndexer(RuntimeEnvironment env,
                     p.setName(name);
                     p.completeWithDefaults();
                     projects.put(name, p);
-                } else if (!name.startsWith(".") && file.isDirectory()) {
+                } else if (file.isDirectory()) {
```

Comment:
what about `.` and `..` ?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/io/File.html#listFiles() says:
> an array of File objects is returned, one for each file or directory in the directory. Pathnames denoting the directory itself and the directory's parent directory are not included in the result.